torchacc/core/__init__.py

```diff
@@ -31,3 +31,14 @@ def fetch_gradients(optimizer):
                     if isinstance(p, torch.Tensor) and p.grad is not None:
                         gradients.append(p.grad.data)
     return gradients
+
+
+def sync(wait: bool = False):
```

Comment:
Can we add some descriptions of these core APIs in the document?

Reply:
Sphinx will generate documentation for all API interfaces. However, we could add a specific document to explain the mechanism behind torchacc. This document can be added in a separate PR.